crm_adjust_all_lead/__openerp__.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2016 Rooms For (Hong Kong) Limited T/A OSCG
+#    <https://www.odoo-asia.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    "name": "crm_adjust_all_lead",
+    "version": "8.0.1.0.0",
+    "author": "Rooms For (Hong Kong) Limited NONO",
+    "license": "AGPL-3",
+    "description": """
+    """,
+    "category": "Sales",
+    "depends": [
+        'crm',
+    ],
+    "data": [
+        'crm_adjust_all_lead/crm_security.xml'

```

Comment:
Put the file in `security` directory.  Follow the structure of this template: https://github.com/OCA/maintainer-tools/tree/master/template/module